Ginger/GingerCore/Resources/GingerHTMLHelper.js

```diff
@@ -1271,6 +1275,13 @@ function define_GingerLib() {
         return pl;
     }
 
+    GingerLib.GetAttributeValue = function (el,attribute) {
+        var pl = new GingerPayLoad("Attribute Value");
+        pl.AddValueString(el.getAttribute(attribute).replace(/GingerHighlight/g,''));
```

Comment:
if el.getAttribute will return null then .replace will throw exception. This will break the payload response. Please add null check to avoid exception